es/readers-guide.md

```diff
@@ -18,12 +18,12 @@ Esta sección explica por qué es importante seguir los requisitos para personas
 
 Esta sección gestiona expectativas a través de la explicación de requisitos que el criterio no cumplirá.
 
-Esto ayuda:
+Esto ayuda a:
 
-* con una correcta aplicación del Estándar
-* a asegurarse de que no aparecen questiones improvistas
+* Aplicar de manera correcta el Estándar.
+* Asegurarse de que surjan imprevistos.
```

Comment:
Good suggestion although capital letters and punctuation like the original Standard, although I like your suggestion